.travis.yml

```diff
@@ -4,6 +4,7 @@ dist: trusty
 env:
```

Comment:
it would be nice to know the difference between 5.15

Reply:
added note to cmt msg